lib/presentation/views/notification/notification_info.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class NotificationInfoView extends StatelessWidget {
+  final String message;
+
+  const NotificationInfoView({Key? key, this.message = '\n \n \n blank'})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Scaffold can be removed. Could you also remove it from `loading.dart`?

Reply:
done